src/quick-lint-js/fe/parse-statement.cpp

```diff
@@ -4436,14 +4458,20 @@ void Parser::parse_and_visit_if(Parse_Visitor_Base &v) {
       QLJS_PARSER_UNIMPLEMENTED();
     }
 
+    if (!allow_several_statements && this->peek().type != Token_Type::kw_else &&
+        this->peek().indent_level > keyword_token.indent_level) {
+      this->diag_reporter_->report(Diag_Misleading_If_Or_Else_Body_Indentation{
+          .if_or_else_span = keyword_token.span()});
+    }
```

Comment:
This code reports diagnostics for the following code:
```javascript
// misleading indentation after 'if' body [E0443]
if (true) console.log();
 console.log();

function f() {
 if (true) console.log();
 // misleading indentation after 'if' body [E0443]
 else console.log()
  console.log();
}

```
Was this intentional? If so, I think it deserves a test to codify the behavior.